pkg/gameservers/controller.go

```diff
@@ -474,6 +477,35 @@ func (c *Controller) syncGameServerPortAllocationState(ctx context.Context, gs *
 	return gs, nil
 }
 
+// Allocate gameserver ports ( containerports ) for dynamic protocols such as TCPUDP
+// TODO: Going forward, this needs to be enhanced for other kinds of dynamic protocols
```

Comment:
Not sure I follow this TODO? I feel like the comment should just be something like "Creates duplicate ports for handling TCPUDP protocol with static port policies" - or something simple like that? There isn't dynamic protocols in this?